.github/workflows/ci.yml

```diff
@@ -16,14 +16,14 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.8
-      - run: pip install flake8
-      - run: flake8 tests xtarfile
+      - run: python -m pip install flake8
+      - run: flake8 --ignore=E501,F401,E722,E712,E221 tests xtarfile
```

Comment:
Instead of ignoring E501, I would suggest to set a longer line length in `setup.cfg`:

```
[flake8]
max-line-length = 120
```

This will mean we'll at least enforce some sort of sensible upper limit.

As for F401, E722, E712 and E221, I would suggest to disable them where appropriate with inline comments as opposed to globally as otherwise we may be hiding real issues in the future.

Reply:
Fixed in commit 458e8a8